frontend/src/components/CountdownDigital.tsx

```diff
@@ -1,72 +1,131 @@
-    ((date.getTime() - new Date().getTime()) %
-      (MILLIS_SECOND * SECONDS_MINUTE * MINUTES_HOUR * HOURS_DAY)) /
-      (MILLIS_SECOND * SECONDS_MINUTE * MINUTES_HOUR),
+    ((timeDiff) % (thousand * sixty * sixty * hoursInDay)) /
+      (thousand * sixty * sixty),
   );
   const minutes = Math.floor(
-    ((date.getTime() - new Date().getTime()) %
-      (MILLIS_SECOND * SECONDS_MINUTE * MINUTES_HOUR)) /
-      (MILLIS_SECOND * SECONDS_MINUTE),
+    ((timeDiff) % (thousand * sixty * sixty)) / (thousand * sixty),
   );
 
+  return { days, hours, minutes };
+};
+
+interface CountdownDigitalProps {
+  date: Date;
+  title?: string;
+}
+
+const CountdownDigital: React.FC<CountdownDigitalProps> = ({
+  date,
+}) => {
+  const currentTime = new Date();
+  const [count, setCount] = useState(0);
+
   useEffect(() => {
-    const interval = setInterval(() => {
-      void queryClient.refetchQueries({
-        queryKey: buildKey(nextTournamentFactory.queryKey, { episodeId }),
-      });
-    }, MILLIS_SECOND * SECONDS_MINUTE);
+    const intervalId = setInterval(() => {
+      setCount((prevCount) => prevCount + 1);
+    }, thousand * countDownRefreshRate);
     return () => {
-      clearInterval(interval);
+      clearInterval(intervalId);
     };
   }, []);
 
-  if (dateHasPassed) return <div>TODO all zeros</div>;
+  const countdownInfo = useMemo(() => computeTimeDiff(currentTime, date), [count]);
+  const progressCircleSize = 140;
+  const percentageDays = ((sixty-countdownInfo.days)/sixty)*fullPercentage;
+  const percentageHours = ((hoursInDay-countdownInfo.hours)/hoursInDay)*fullPercentage;
+  const percentageMins = ((sixty-countdownInfo.minutes)/sixty)*fullPercentage;
+  const countTxtSize = "3xl";
+  const labelTxtSize = "l";
+
+
   return (
-    <div className="flex max-w-max flex-col gap-1">
-      <span className="text-2xl font-bold">{title}</span>
-      <div className="grid grid-cols-11 grid-rows-2 content-center justify-center gap-2">
-        <div className="col-span-3 self-end text-4xl font-bold">{days}</div>
-        <div className="text-4xl font-bold">:</div>
-        <div className="col-span-3 self-end text-4xl font-bold">{hours}</div>
-        <div className="text-4xl font-bold">:</div>
-        <div className="col-span-3 self-end text-4xl font-bold">{minutes}</div>
-        <div className="col-span-3 self-start">Days</div>
-        <div />
-        <div className="col-span-3 self-start">Hours</div>
-        <div />
-        <div className="col-span-3 self-start">Minutes</div>
+    <div className={`flex flex-row max-w-max justify-evenly gap-8`}>
+        <div className="relative">
+          {ProgressCircle({sqSize: progressCircleSize, percentage: percentageDays})}
+
+          <div className={`absolute top-${progressCircleSize / 2} left-${progressCircleSize / 2} top-1/2 start-1/2 transform -translate-x-1/2 -translate-y-1/2 text-center`}>
+            <span className={`font-bold text-${countTxtSize}`}>{countdownInfo.days}</span>
```

Comment:
nit: looks like we are repeating this classname and others in a few different places... I think it would be good to refactor these into constants that we can reuse and keep them consistent :sunglasses: